build.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
-if [ -d "dmd2" ]; then
```

Comment:
This should stay in in some form (basically, if someone already has a compatible DMD compiler installed, I don't want to force them to run dependencies.sh.)